### PR TITLE
Include small details about formatting code

### PR DESCRIPTION
**Pull request by nilsmartel**

### CODE REVIEW

File: docs/src/userguide/formatter.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+print(f'(2))
+
+open("myfile.txt", "w") do io
+	write(io, "Hello world!")
+end;
+```
+
+It's the very same code, though now it's much easier to read.
+
+In order to format your code press `shift + cmd|windows + p` to bring up the command palette and search for `Format Document`
```
Comment (pfitzseb) [inline docs/src/userguide/formatter.md:27]: I think `Ctrl-Shift-I` is the default keybinding for formating a document, right?

File: docs/src/userguide/formatter.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+<img width="741" alt="grafik" src="https://user-images.githubusercontent.com/28377948/135749171-c6708afb-0f08-4268-b041-d24de4226e7f.png">
```
Comment (pfitzseb) [inline docs/src/userguide/formatter.md:29]: This needs a [`@raw` block](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@raw-format-block), no?

**Review by pfitzseb — COMMENTED** [review]
Thanks for the contribution! Left two small comments inline.

File: docs/src/userguide/formatter.md (hunk 1 above)
Comment (nilsmartel) [reply docs/src/userguide/formatter.md:27]: I agree, though it did not work for me, and I couldnt figure out why (it was even shown to be that combination for me, oddly enough).

What would you say is the right thing to do in the docs then?

File: docs/src/userguide/formatter.md (hunk 2 above)
Comment (nilsmartel) [reply docs/src/userguide/formatter.md:30]: glad you caught it

**Review by pfitzseb — APPROVED** [review]

**Pull request merged**